tests/test_apply_variants.py

```diff
@@ -555,45 +579,48 @@ def test_reference_alignment_interior_anchor(self):
 
     def test_reference_alignment_other_anchor_cases(self):
         genome37 = MiniGenome('test_genome')
-        variants = [Variant.from_string("chr1:11:GTA:ATG", genome37)]
 
-        # Test anchor==start
+        # Test anchor==start with substitution
+        substitution_variant = [Variant.from_string("chr1:11:GTA:ATG", genome37)]
         interval = Interval('chr1', '+', 5, 15, genome37, 5)
-        reference_alignment = apply_variants(genome37.dna, variants, interval, reference_alignment=True)[1]
+        reference_alignment = apply_variants(genome37.dna, substitution_variant, interval, reference_alignment=True)[1]
 
         self.assertEqual(reference_alignment, [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
 
-        # Test anchor==start
+        # Test anchor==end with substitution
+        substitution_variant = [Variant.from_string("chr1:11:GTA:ATG", genome37)]
```

Comment:
why is substitution_variant redeclared for every test if its value doesn't change?
I appreciate renaming to insertion_variant where relevant, but no need to redeclare before every case.